app/scripts/components/app/edit.js

```diff
@@ -66,6 +67,35 @@ class EditRecord extends React.Component {
     }
   }
 
+  // static getDerivedStateFromProps (nextProps, prevState) {
```

Comment:
Seems like we can remove this

Reply:
oh no, that was a previous attempt. I kept it in just in case you wanted to see that I had tried doing it the other way. But I can remove it.